modules/react/button/stories/visual-testing/stories_TextButton.tsx

```diff
@@ -9,30 +9,31 @@ import {
   customColorTheme,
 } from '../../../../../utils/storybook';
 import {playCircleIcon} from '@workday/canvas-system-icons-web';
-import {TextButton} from '../../index';
+import {TextButton} from '@workday/canvas-kit-react/button';
 import {Container, stateTableColumnProps} from './utils';
+import {PartialEmotionCanvasTheme} from '@workday/canvas-kit-react/common';
 
 export default withSnapshotsEnabled({
   title: 'Testing/React/Buttons/Button/Text Button',
   component: TextButton,
 });
 
-export const TextButtonStates = () => (
-  <StaticStates>
+export const TextButtonStates = (props: {theme?: PartialEmotionCanvasTheme}) => (
+  <StaticStates theme={props.theme}>
     <ComponentStatesTable
       rowProps={permutateProps(
         {
           variant: [
-            {value: TextButton.Variant.Default, label: 'Default'},
-            {value: TextButton.Variant.Inverse, label: 'Inverse'},
+            {value: 'text', label: 'Default'},
+            {value: 'inverse', label: 'Inverse'},
           ],
           allCaps: [
             {value: false, label: ''},
             {value: true, label: 'All Caps'},
           ],
           size: [
-            {value: TextButton.Size.Small, label: 'Small'},
-            {value: TextButton.Size.Large, label: 'Large'},
+            {value: 'small', label: 'Small'},
+            {value: 'medium', label: 'Medium'},
```

Comment:
Not 100% sure why the visual tests were testing 'Large' when the component states only `small` and `medium` are allowed. The visual tests look the same except "Medium" is a longer word and sometimes requires more lines to render the label which is a noisy diff.